[ 2025-03-02 10:24:51,886 ] 76 root - INFO - Loading dataset...
[ 2025-03-02 10:24:51,889 ] 100 root - INFO - Applying data transformations...
[ 2025-03-02 10:24:51,889 ] 16 root - INFO - Performing data cleaning...
[ 2025-03-02 10:24:51,897 ] 28 root - INFO - Creating new features...
[ 2025-03-02 10:24:51,904 ] 39 root - INFO - Encoding and standardizing data...
[ 2025-03-02 10:24:52,465 ] 32 root - ERROR - \u274c Error loading object from artifacts/best_model.pkl: No module named '_loss'
[ 2025-03-02 10:24:52,466 ] 33 root - ERROR - \u274c Failed to load model: Error occured in python script name [E:\all projects\Machine Learning Projects\Suppervised Project\Reggration Projects\Big_mart_sales\src\big_mart_sales\utils.py] line number [29] error message[No module named '_loss']
[ 2025-03-02 10:24:52,466 ] 63 root - ERROR - \u274c Error during prediction: Error occured in python script name [E:\all projects\Machine Learning Projects\Suppervised Project\Reggration Projects\Big_mart_sales\src\big_mart_sales\pipelines\prediction_pipeline.py] line number [29] error message[Error occured in python script name [E:\all projects\Machine Learning Projects\Suppervised Project\Reggration Projects\Big_mart_sales\src\big_mart_sales\utils.py] line number [29] error message[No module named '_loss']]
[ 2025-03-02 10:24:52,467 ] 64 root - ERROR - \u274c Prediction failed
